packages/plugins/src/cli/utils.ts

```diff
@@ -1,34 +1,15 @@
 import semver from 'semver'
 
+export const NoNetworkArgsError = `Expected at least one network, but none were supplied.`
+
 export const BothNetworksSpecifiedError = `You must specify either 'testnets' or 'mainnets', but not both.`
 
 export const ConfirmAndDryRunError =
   '--confirm and --dry-run cannot be used together'
 
-export const getInvalidNetworksArgumentError = (arg: string | Array<string>) =>
-  `Invalid values:\n  Argument: networks, Given: "${arg}", Choices: "testnets", "mainnets"`
-
-export const coerceNetworks = (
-  arg: string | Array<string>
-): 'testnets' | 'mainnets' => {
-  // Check if `arg` is an array and has both 'mainnets' and 'testnets'.
-  if (
-    Array.isArray(arg) &&
-    arg.length === 2 &&
-    arg.includes('testnets') &&
-    arg.includes('mainnets')
-  ) {
-    throw new Error(BothNetworksSpecifiedError)
-  }
```

Comment:
Moved this elsewhere since we no longer coerce the yargs args into `testnets`/`mainnets`